packages/media/src/preloadImage.js

```diff
@@ -19,11 +19,13 @@
  *
  * @param {string} src Image source.
  * @param {string} [srcset] Image source set.
+ * @param {number|string} [width] Image width.
+ * @param {number|string} [height] Image height.
  * @return {Promise} Image object.
  */
-const preloadImage = (src, srcset = undefined) => {
+const preloadImage = (src, srcset = undefined, width, height) => {
```

Comment:
now that `preloadImage` has 4 params getting passed this would be more readable if it's an object: 
`const preloadImage = ({src, srcset = undefined, width, height}) => {` this helps me know why you're passing `undefined` on the background contrast checklist.

Reply:
Fixed,